subsys/bluetooth/host/gatt.c

```diff
@@ -274,6 +275,28 @@ static void sc_process(struct k_work *work)
 	atomic_set_bit(sc->flags, SC_INDICATE_PENDING);
 }
 
+static struct gatt_ccc_store {
+	struct bt_conn *conn;
+	atomic_t scheduled;
+	struct k_delayed_work work;
+} gatt_ccc_store;
```

Comment:
Is it intentional that this variable exists (i.e. consumes memory) even if `CONFIG_BT_SETTINGS_STORE_ON_WRITE` is not enabled?

Reply:
The variable should only be created if the Kconfig option is enabled. Thanks, fixed.